components/script/dom/webglrenderingcontext.rs

```diff
@@ -231,12 +269,12 @@ impl WebGLRenderingContext {
 
         result.map(|ctx_data| {
             WebGLRenderingContext {
+                limits: WebGLRenderingContextLimits::new(&ctx_data.sender),
```

Comment:
Get the WebGLRenderingContextLimits from ctx_data as mentioned in the other comment